rewriter/GenCallAsm.cpp

```diff
@@ -280,21 +280,309 @@ static std::string sig_string(const CAbiSignature &sig,
   return ss.str();
 }
 
+static void call_string(
```

Comment:
nit: Maybe `emit_fn_call` or `emit_call_target_fn`?

Reply:
Addressed this in the reshuffle.